### PR TITLE
404

### PR DESCRIPTION
**Pull request by Thasbia**

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        border-radius: 5px;
+        transition: background-color 0.3s ease;
+        font-weight: 600;
+      }
+
+      .button:hover {
+        background-color: #ea580c;
+      }
+      footer {
+        height: 100vh; /* Set the height to 100% of the viewport height */
```
Comment (HaydenBS93) [inline 404.html:57]: You can remove the comment on this

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+          </div>
+          <div class="centered">
+            <h1 class="heading">404</h1>
+          </div>
+          <div class="centered">
+            <h2 class="subheading">Page not found</h2>
+          </div>
+          <div class="back-to-home">
+            <a href="/" class="button">Back to Home</a>
+          </div>
```
Comment (HaydenBS93) [inline 404.html:118]: Do you think you could use the <button> tag for this within the <a> tag?
Also needs to link back to the home page.

File: 404.html (hunk 1 above)
Comment (Thasbia) [reply 404.html:57]: Completed Boss man

File: 404.html (hunk 2 above)
Comment (Thasbia) [reply 404.html:118]: Completed Hamster H

**Review by HaydenBS93 — APPROVED** [review]
All good 🥇

**Pull request merged**